packages/monitor-v2/src/monitor-polymarket/abi/binaryAdapter.json

```diff
@@ -0,0 +1,332 @@
+[
```

Comment:
not ideal that we have to do this...would be great if we could import it from polymarkets exported sdk or something equivalent to this.

Reply:
I haven't found anything in their sdks. This abi's allow us to read directly on-chain